examples/advanced/sklearn-svm/job_configs/sklearn_svm_base/app/custom/svm_assembler.py

```diff
@@ -0,0 +1,63 @@
+# Copyright (c) 2021-2022, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# We will move to this app_common when it gets matured
+from typing import Dict
+
+import numpy as np
+from sklearn.svm import SVC
+
+from nvflare.apis.dxo import DataKind
+from nvflare.app_common.aggregators.assembler import Assembler
+
+
+class SVMAssembler(Assembler):
+    def __init__(self, kernel):
+        super().__init__(data_kind=DataKind.WEIGHTS)
+        # Record the global support vectors
+        # so that only 1 round of training is performed
+        self.support_x = None
+        self.support_y = None
+        self.kernel = kernel
+
+    def get_model_params(self, data: dict):
+        return {"support_x": data["support_x"], "support_y": data["support_y"]}
+
+    def assemble(self, current_round: int, data: Dict[str, dict]) -> dict:
+        if current_round == 0:
+            # First round, collect all support vectors from clients
+            support_x = []
+            support_y = []
+            for client in self.collector:
+                client_model = self.collector[client]
+                support_x.append(client_model["support_x"])
+                support_y.append(client_model["support_y"])
+            global_x = np.concatenate(support_x)
+            global_y = np.concatenate(support_y)
+            # perform one round of SVM to produce global model
+            svm_global = SVC(kernel=self.kernel)
+            svm_global.fit(global_x, global_y)
+            # get global support vectors
+            index = svm_global.support_
+            self.support_x = global_x[index]
+            self.support_y = global_y[index]
+        # The following rounds, if any, directly returns the retained record
+        # No further training
+        params = {"support_x": self.support_x, "support_y": self.support_y}
+        return params
```

Comment:
instead of "following rounds directly returns the retained records",

would it be better to show "warning messages" and stop the learning?
otherwise there might be people setting rounds to 100 (carelessly) and later figure out only first round is needed.

Reply:
handled at learner side